### PR TITLE
emails on publishing and rejecting

### PR DESCRIPTION
**Pull request by ksiedlarek**

### CODE REVIEW

File: jobs/admin.py [hunk 1]
```diff
@@ -29,14 +28,16 @@ def send_status_update_job_offer(modeladmin, request, queryset):
...
                 })
         )
         message_html = get_template(
             'jobs/email_templates/status.html').render(
                 Context({
                     'status': item.get_review_status_display(),
-                    'option': option
+                    'option': option,
+                    'reviewers_comment': item.reviewers_comment,
                 })
```
Comment (rebkwok) [inline jobs/admin.py]: move `Context({...})` out into a variable so you don't need to duplicate it in the plain and html get_template

File: jobs/admin.py [hunk 2]
```diff
@@ -73,14 +74,16 @@ def send_status_update_meetup(modeladmin, request, queryset):
...
         )
         message_html = message_plain = get_template(
             'jobs/email_templates/status.html').render(
                 Context({
                     'status': item.get_review_status_display(),
-                    'option': option
+                    'option': option,
+                    'reviewers_comment': item.reviewers_comment,
                 })
```
Comment (rebkwok) [inline jobs/admin.py]: same as above - move Context out to a variable

File: jobs/models.py [hunk 3]
```diff
@@ -88,24 +93,88 @@ def accept(self):
...
+                    'reviewers_comment': self.reviewers_comment,
+                })
+            )
+        message_html = get_template(
+            'jobs/email_templates/status.html').render(
+            Context({
+                'status': self.get_review_status_display(),
+                'option': option,
+                'reviewers_comment': self.reviewers_comment,
+            })
```
Comment (rebkwok) [inline jobs/models.py]: move Context to a variable

File: jobs/models.py [hunk 4]
```diff
@@ -88,24 +93,88 @@ def accept(self):
...
```
Comment (rebkwok) [inline jobs/models.py]: move Context to a variable

File: jobs/tests/test_models.py [hunk 5]
```diff
@@ -86,23 +86,23 @@ def test_accept_for_jobs_not_under_review(self):
...
         """Tests the reject method for jobs in the PUBLISHED state"""
-        self.job_published.reject()
+        self.job_published.reject(option='job')
         self.assertTrue(self.job_published.review_status == Job.REJECTED)
         self.assertFalse(self.job_published.published_date)
 
     def test_reject_for_jobs_in_wrong_state(self):
         """Tests the reject method for jobs not in the wrong state"""
-        self.assertRaises(AssertionError, self.job_open.reject)
+        self.assertRaises(AssertionError, self.job_open.reject, option='job')
```
Comment (rebkwok) [inline jobs/tests/test_models.py]: `self.job_open.reject(option='job')` ?

File: jobs/tests/test_models.py [hunk 6]
```diff
@@ -116,11 +116,11 @@ def test_restore_for_jobs_not_rejected(self):
     def test_publish_not_ready_to_publish(self):
         """Attempts to publish jobs which are not in the READY_TO_PUBLISH state,
         it should results in an assertion error"""
-        self.assertRaises(AssertionError,  self.job_open.publish)
+        self.assertRaises(AssertionError,  self.job_open.publish, option='job')
```
Comment (rebkwok) [inline jobs/tests/test_models.py]: `self.job_open.publish(option='job')` ?

File: jobs/tests/test_models.py [hunk 7]
```diff
@@ -142,9 +142,9 @@ def test_publish_with_custom_expiration_date(self):
     def test_publish_twice_in_a_row(self):
         """It is not possible to publish a job offer twice"""
         self.assertFalse(self.job_ready_no_exp_date.published_date)
-        self.job_ready_no_exp_date.publish()
+        self.job_ready_no_exp_date.publish(option='job')
         self.assertTrue(self.job_ready_no_exp_date.published_date, "Job has no published date.")
-        self.assertRaises(AssertionError, self.job_ready_no_exp_date.publish)
+        self.assertRaises(AssertionError, self.job_ready_no_exp_date.publish, option='job')
```
Comment (rebkwok) [inline jobs/tests/test_models.py]: `self.job_ready_no_exp_date.publish(option='job')` ?

File: jobs/tests/test_models.py [hunk 8]
```diff
@@ -168,11 +168,11 @@ def setUp(self):
     def test_publish_without_ready_to_publish(self):
         """Attempts to publish meetups with ready_to_published=False
         should results in an assertion error"""
-        self.assertRaises(AssertionError,  self.meetup_not_ready.publish)
+        self.assertRaises(AssertionError,  self.meetup_not_ready.publish, option='job')
```
Comment (rebkwok) [inline jobs/tests/test_models.py]: `self.meetup_not_ready.publish(option='job')` ?

File: jobs/views.py [hunk 9]
```diff
@@ -94,11 +94,15 @@ def create_job(request):
             subject = '{0} was submitted succesfully.'.format(new_job)
             message_plain = get_template(
             'jobs/email_templates/community_confirm.txt').render(
-                Context({ 'option': new_job })
+                Context({
+                    'option': new_job
+                })
```
Comment (rebkwok) [inline jobs/views.py]: No need to split this across lines, just keep the final parenthesis on the next line

File: templates/jobs/email_templates/status.txt [hunk 10]
```diff
@@ -2,5 +2,10 @@ Hello there!
 
 This is an automatically generated email about your {{ option }}. Current status is: {{ status }}.
 
+{% if reviewers_comment %}
+    This is your feedback:
+    {{ reviewers_comment|linebreaks }}
+{% endif %}
```
Comment (rebkwok) [inline templates/jobs/email_templates/status.txt:10]: Review with maildump if you can to see how this formats.  I think text emails will have extra line breaks where the template tags are, and will format with the indents.  Just make sure it looks how you want it to.

**Comment by rebkwok** [comment]
Also will need conflicts resolved before merging

**Comment by ksiedlarek** [comment]
emails are not pretty yet, but I've checked with maildump (great tool :)), it's not horrible.

**Comment by rebkwok** [comment]
:+1:

**Pull request merged**